mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -409,6 +409,90 @@ trait Completions { this: MetalsGlobal =>
         NoneCompletion
     }
   }
+
+  private def isComment(str: String, pos: Position): Boolean = {
```

Comment:
So we discussed this with @Giggiux and there seems to be no better way of finding a comment and it looks to be on the heavy side. The compiler doesn't seem to over a sensible way. I am thinking that we should leave it as is and close the issue as `wond't fix` - it's a small enough issue that shouldn't bother users in most cases. What do you think @ckipp01 ?

Reply:
If we don't want to tackle it now, that's fine, but i wouldn't close it with `won't fix`. I think it's a reasonable expectation that in this situation you wouldn't get completions offered to you. For example I just checked IntelliJ and it doesn't offer completions like this in comments, so I think a popup that offers completion options like `trait` etc in a comment is really off putting.

Reply:
Hmm.. in VS Code this doesn't seem to be a big problem, but I wonder whether it is only the keyword completions? Are you getting something else too? I think we can optimize it a bit in keywords :thinking:

Reply:
I think the reason I notice it so much is that every time use the scaladoc completion, after the completion finishes the next thing I see is this:

<img width="418" alt="Screenshot 2021-05-30 at 10 05 21" src="https://user-images.githubusercontent.com/13974112/120097020-a375e100-c12e-11eb-88f8-837b5a2f54b3.png">

It's only these keywords that are every offered.